torchvision/ops/boxes.py

```diff
@@ -114,7 +114,12 @@ def _batched_nms_vanilla(
 
 def remove_small_boxes(boxes: Tensor, min_size: float) -> Tensor:
     """
-    Remove boxes which contains at least one side smaller than min_size.
+    Remove every box from :attr:`boxes` which contains at least one side length
+    that is smaller than :attr:`min_size`.
+
+    .. warning::
```

Comment:
Here and in other places, let's just make this a note instead of a warning.

Reply:
I'm fine with that, I just thought that we might want to slowly deprecate the redundant functions, that were implemented before the `tv_tensors.BoundingBoxes` were a thing. Starting to get users to use the newer functions.

But that is a discussion for another time I would assume.